lib/server_health_check.rb

```diff
@@ -25,13 +25,17 @@ def redis!(host: nil, port: 6379)
 
   def active_record!
     begin
-      ActiveRecord::Base.connection
-      @results[:active_record] = OK
-      true
+      if ActiveRecord::Base.connection_pool.with_connection { |connection| connection.active? }
+        @results[:active_record] = OK
+        true
+      else
+        @results[:active_record] = "Failed: unable to connect to database"
+        false
+      end
     rescue NameError
```

Comment:
Is the idea that we want to warn the client that AR isn't defined?

I was assuming that if AR isn't defined, it means we're using this gem in code that doesn't use AR and there's nothing to worry about.  This was probably an incorrect assumption, though it seems strange to me that this would be the right place to do that.